.travis.yml

```diff
@@ -6,10 +6,22 @@ branches:
   only:
     - master
     - develop
+
+os:
+  - osx
```

Comment:
We should really remove the osx, since it puts our build in a queue to wait for an available osx machine and it takes about 30 minutes :(